app/Models/MplsSdp.php

```diff
@@ -30,7 +30,7 @@ class MplsSdp extends Model implements Keyable
 
     /**
      * Get a string that can identify a unique instance of this model
-     * @return string
+     * @return int
```

Comment:
Actually, this should be a string :D

Reply:
Maybe change the interface to int|string

Reply:
https://github.com/librenms/librenms/pull/12665/files/c0fee82100430e50da3216a0ad4802ed1f1e1220#diff-ec2ca4badb133868081b56ffbca8d1ca48579a7953d23825eecf4fb9380d53d5R31

;)